### PR TITLE
Add engine.isParticipant()

### PR DESCRIPTION
**Pull request by GreenAsJade**
... so that we have a single way of asking if someone is a participant.

### CODE REVIEW

**Comment by benjaminpjones** [comment]
lol look what i found.. 😅 

https://github.com/online-go/goban/blob/c6c9f42d566497a4a6b0247384f759b6b44c0f7c/src/GobanCore.ts#L3485-L3490

**Comment by GreenAsJade** [comment]
🤦‍♀️    Darn!   I searched for isPlayer ...

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
     "name": "goban",
-    "version": "0.5.75",
+    "version": "0.5.76",
```
Comment (anoek) [inline package.json:3]: FTR you don't need to bump this, it happens automatically on release

**Comment by anoek** [comment]
Thanks for the improvement :)

**Pull request merged**